src/cloudant/replicator.py

```diff
@@ -101,11 +100,9 @@ def create_replication(self, source_db=None, target_db=None,
 
         # add user context delegation
 
-        if not data.get('user_ctx'):
-            if target_db and target_db.admin_party:
-                pass  # noop - not required for admin party mode
-            elif self.database.creds and self.database.creds.get('user_ctx'):
-                data['user_ctx'] = self.database.creds['user_ctx']
+        if not data.get('user_ctx') and self.database.creds and \
```

Comment:
Is there a case where `user_ctx` doesn't exist in `data` and `database.creds`?

Reply:
yeah, admin party mode.